app/jobs/inventory_host_updated_job.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# Updates the host according to the inventory definition
+class InventoryHostUpdatedJob
+  include Sidekiq::Worker
+  include RecordNotFound
+
+  def perform(message)
+    if message['host'] && message['host']['display_name']
+      rescue_not_found do
+        Host.find_by!(
+          id: message['host']['id']
+        )&.update(name: message['host']['display_name'])
```

Comment:
This safe operator `&.` won't ever be used because `find_by!` raises an exception, never returning `nil`.